### DISCUSSION TITLE
AI輔助診斷降失明風險

### DISCUSSION THREAD

**Discussion by bww0401** [1101-GS3073-C/Python-Community-GS3073-C#494] [科技新知]
台灣將近250萬人口患有糖尿病，但全國眼科醫生只有2033人，用這套AI輔助視網膜病變診斷，可以減少很多醫療的成本與負擔!
新聞網址: https://tw.news.yahoo.com/%E6%96%B0%E7%A7%91%E6%8A%80%E5%A5%BD%E5%A8%81-%E8%9C%98%E8%9B%9B%E7%B5%B2%E6%B8%AC%E8%A1%80%E7%B3%96-ai%E8%BC%94%E5%8A%A9%E8%A8%BA%E6%96%B7%E9%99%8D%E5%A4%B1%E6%98%8E%E9%A2%A8%E9%9A%AA-%E7%94%9C%E8%9C%9C%E5%8D%B1%E6%A9%9F%E7%B3%96%E5%B0%BF%E7%97%85%E4%B9%8B%E5%B3%B6-145532315.html

**Comment by lx3211** [comment]
雖然跟AI沒關係，但是文章裡提到用蜘蛛絲測血糖的想法真的很特別誒，不知道怎麼想才會想到要用蜘蛛絲

**Comment by manipulateed** [comment]
這類的ai 輔助能大大減緩人力資源等等不足的問題，而文中蜘蛛絲感測血糖的研究更是讓人大開眼界，彷彿置身於平常的環境中，有更多事情等著我們去發現

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by xCk27x** [comment]
【研發出了這款光纖糖度感測器，未來可以適用於長期放在人體裡面，24小時都可以監測血糖】
文章裡提到的這個想法很不錯ㄟ
對糖尿病患者來說可以省去很多麻煩
還能增加監控頻率
但如何縮小檢測儀的體積是個挑戰

**Comment by williamyen0623** [comment]
最近在系上的專題展有看到類似的作品
是訓練AI模型 達到偵測視網膜是否病變的功能
近年來AI用在醫療輔助上真的越來越多
希望往後業界人士能加快發展 讓大家都能健健康康～

**Comment by BingRong1217** [comment]
AI在醫療方面可以節省很多人力

**Comment by skyling0299** [comment]
如果能用AI追蹤病變範圍對醫療也有幫助

**Comment by Ben6uo** [comment]
這樣可以大量降低醫師們的負擔

**Comment by limitx0** [comment]
蜘蛛絲真的是很神奇的材料

**Comment by Aerofe** [comment]
AI與醫療的結合真的讓病患能夠更早預防疾病發生，但文中的蜘蛛絲測血糖真的是個創新的想法

**Comment by Raywei0214** [comment]
我們普遍認為，若能理解透徹核心原理，對其就有了一定的了解程度。如果此時我們選擇忽略失明，那後果可想而知。

**Comment by larry40305** [comment]
AI輔助醫療行為的越來越廣泛，這樣可以大幅減少醫療人力
